utils/dom.js

```diff
@@ -10,16 +10,30 @@ import tinymce from 'tinymce';
 const { getComputedStyle, DOMRect } = window;
 const { TEXT_NODE, ELEMENT_NODE } = window.Node;
 
+/**
+ * Returns true if there is a selection and if the selection is collapsed, or
+ * false otherwise.
+ *
+ * @return {boolean} Whether there is a collapsed selection.
+ */
+export function hasCollapsedSelection() {
+	const selection = window.getSelection();
```

Comment:
There's also `selection.isCollapsed` which might work too. https://developer.mozilla.org/en-US/docs/Web/API/Selection

Reply:
>There's also `selection.isCollapsed` which might work too.

At that point, we may as well not even have this function, since `window.getSelection().collapsed` is barely more verbose.

Reply:
Yeah :)

```js
const { getSelection } = window;

getSelection().isCollapsed;
```